backend/src/app.js

```diff
@@ -3,9 +3,16 @@
 const AutoLoad = require('fastify-autoload');
 const path = require('path');
 
+require('dotenv').config();
+const { DOMAIN, FRONTEND_PORT } = process.env;
+
 module.exports = async function (fastify, opts) {
+  fastify.register(require('fastify-cors'), {
```

Comment:
inline require should trigger linter, anyway it's not very easy to support where deps inlined somewhere in logic